scripts/scripts-1.js

```diff
@@ -0,0 +1,90 @@
+for (let i = 10; i < 26; i += 0.5) {
+
+  console.log(i);
+}
+
+const number = +prompt("Введіть число"); {
+
+  if (number % 2 !== 0 && number % 3 !== 0) {
+
+    console.log(`${number} просте число`);
+  }
+
+  else {
+    console.log(`${number} складене число`);
+  }
+}
+
+localStorage.setItem("number", number);
+
+discount = +prompt("Сума вашої покупки становить:");
+
+if (discount <= 100) {
+
+  alert(`Вітаю, ваша знижка 3%!`);
+}
+
+else if (discount <= 200) {
+
+  alert(`Вітаю, ваша знижка 5%!`);
+}
+
+else if (discount > 200) {
+
+  alert(`Вітаю, ваша знижка 7%!`);
+}
+
+localStorage.setItem("discount", discount);
+
+//таблиця множення
+
+for (i = 1; i <= 10; i++) {
+
+  console.log(`Множимо на "${i}"`);
+
+  for (j = 1; j <= 10; j++) {
+
+    console.log(`${i} * ${j} = ${i * j}`);
+  }
+}
+
+const oneDollar = 36.76;
+
+for (let i = 10; i <= 100; i += 10) {
+
+  console.log(`${i} USD = ${oneDollar * i} UAH`);
+
+  // console.log(`${i} USD = ${Math.round(oneDollar) * i} UAH`); Тут я округлила до найближчого цілого і вийшли гарні числа:)
+}
+
+/* із завданням №7 у мене вийшла серйозна запара, на жаль... Я ніяк не можу додуматись, щоб гарно згребсти усі цифри і розкласти іх у консольці*/
+
+
+do {
```

Comment:
So, the task was to ask a user to enter 15 numbers and then calculate positive/negative count, odd/even count and zeros
First of all, we should initialize variables for numbers

```
let positiveNumbers = 0;
let negativeNumbers = 0;
let zero = 0;
let evenNumbers = 0;
let oddNumbers = 0;
```

Then let's get the numbers:
**option 1**
 Ask the user to enter all numbers in one prompt and make an array of the numbers: 
```
const numbers = prompt('Please enter fifteen numbers divided by space!');
const arrayFromNumbers = numbers.split(' ');
```
After that, we should sort the numbers by initialized params, lets's use `for` loop
```
for (let i = 0; i < arrayOfNumbers.length; i++) {
const currentElement = Number(arrayNumbers[i]);
    if (currentNumber > 0) {
        positiveNumbers++
    }
    if (currentNumber < 0) {
        negativeNumbers++
    }
    if (currentNumber === 0) {
        zero++
    }
    if (currentNumber % 2 === 0) {
        evenNumbers++
    }
    if (currentNumber % 2 !== 0) {
        oddNumbers++
    }
}

```

**option 2**
Ask the user to enter numbers one by one, and then operate with the numbers inside the loop

```
for (let i = 0; i <= 15; i++) {
    let currentNumber = +prompt(`enter number ${i+1}`);
 if (currentNumber > 0) {
        positiveNumbers++
    }
    if (currentNumber < 0) {
        negativeNumbers++
    }
    if (currentNumber === 0) {
        zero++
    }
    if (currentNumber % 2 === 0) {
        evenNumbers++
    }
    if (currentNumber % 2 !== 0) {
        oddNumbers++
    }
}
```

Finally, displaying our calculation in the console

```
console.log(`There are ${positiveNumbers} positive numbers`);
console.log(`There are ${negativeNumbers} negative numbers`);
console.log(`There are ${zero} zero numbers`);
console.log(`There are ${evenNumbers} even numbers`);
console.log(`There are ${oddNumbers} odd numbers`);
```

Hope it was helpful, let me know if you have any questions